db/db_impl/db_impl.cc

```diff
@@ -1162,7 +1216,10 @@ Status DBImpl::ApplyReplicationLogRecord(ReplicationLogRecord record,
       case ReplicationLogRecord::kManifestWrite: {
         Slice src = record.contents;
         autovector<VersionEdit> edits;
-        DeserializeReplicationLogManifestWrite(&src, &edits);
+        s = DeserializeReplicationLogManifestWrite(&src, &edits);
+        if (!s.ok()) {
+            break;
```

Comment:
does `break vs return` here matter? We cleanup job context and `writeThread. ExitUnbatched` after break. But I guess as long as return status is not ok, the follower shard will be reopened so it does't matter?